src/Homework.java

```diff
@@ -1,5 +1,45 @@
+import java.time.LocalDate;
+
 public class Homework {
-    public static void main (String[] args) {
-        System.out.println("Hello world!");
+    public static void main(String[] args) {
+        int year = 2024;
+        task1(year);
+        int clientOC = 0;
+        int clientDeviceYear = LocalDate.now().getYear();
+        task2(clientOC,clientDeviceYear);
+        int deliveryDistance = 71;
+        task3(deliveryDistance);
+
     }
-}
\ No newline at end of file
+
+    public static void task1(int year) {
```

Comment:
```suggestion
    public static void printIsLeapYearOrNot(int year) {
```
методы надо называть так, чтобы по названию было понятно, что метод делает (см. внимательнее задание, там прям написан этот пункт в правилах)